armi/utils/units.py

```diff
@@ -80,7 +80,7 @@
 MIN_FUEL_HM_MOLES_PER_CC = 1e-10
 
 # More than 10 decimals can create floating point comparison problems in MCNP and DIF3D
-FLOAT_DIMENSION_DECIMALS = 10
+FLOAT_DIMENSION_DECIMALS = 8
 EFFECTIVELY_ZERO = 10.0 ** (-1 * FLOAT_DIMENSION_DECIMALS)
```

Comment:
This `EFFECTIVELY_ZERO` value appears to be used in one of our downstream plugins. The value here will change, but I don't expect the change to have any practical impact.

Reply:
Agree. The `EFFECTIVELY_ZERO` is only used in downstream MCNP plugin.